.github/workflows/publish.yml

```diff
@@ -3,21 +3,33 @@ on:
   push:
     tags:
       - '*'
+  workflow_dispatch:
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
 jobs:
   publish-maven:
     name: Publish to Maven
-    runs-on: ubuntu-latest
+    runs-on: macos-11
```

Comment:
Please, why replacing ubuntu by macos? I think it is slower.

Reply:
You are right! Reverting back to ubuntu